### PR TITLE
PR to merge "list-struct" to the "main"

### PR DESCRIPTION
**Pull request by AliRezaBmeDu**
In this branch I have done the following changes.

- Set up the project with webpack
- Create an index.js file and set an array of some simple to do tasks.
- On page load render the dynamically created list of tasks
- Create a style.css and set rules for the To Do List.
- Fix the linters for html-css-javascript

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @AliRezaBmeDu,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a descriptive readme documentation.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Gambit142 — COMMENTED** [review]
Hi @AliRezaBmeDu 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly list 1-3 features that can be added to this project in your Future Features section 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  checkbox.id = task.index;
+  checkbox.name = task.completed;
+
+  const taskDescription = document.createElement('p');
+  taskDescription.innerText = task.description;
+
+  singleTask.appendChild(checkbox);
+  singleTask.appendChild(taskDescription);
+  listDiv.appendChild(singleTask);
+};
```
Comment (LionRouge1) [inline src/index.js]: - [ ] Nice work populating the HTML template for each task :clap: However it is missing the icons at the right side of the task list. Kindly make sure your list is a clone of the part of the minimalist project captured in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) provided.
### Your design :point_down: 
Screenshot from 2023-07-18 14-55-34
### What it is supposed to look like :point_down: 
Screenshot from 2023-07-18 14-54-33

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @AliRezaBmeDu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job using Webpack to set up the project :1st_place_medal: 
- Nice work iterating over the `todoTasks` array and populates an HTML template for each task :+1: 
- All linters passed :heavy_check_mark: 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @AliRezaBmeDu 
 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

  unnamed


## Highlights
- No linter error :heavy_check_mark: 
- You implemented all suggested changes :heavy_check_mark: 
 
## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**